timesync/peersync/sync.go

```diff
@@ -182,21 +178,23 @@ func (s *Sync) requestHandler(ctx context.Context, buf []byte) []byte {
 // Start background workers.
 func (s *Sync) Start() {
 	s.once.Do(func() {
+		// NOTE(dshulyal) we can't start listening for peers in New because Simular
```

Comment:
`Simular` looks like a typo. What is it?

Reply:
it was Simulator